cookbooks/continental_extension.prm

```diff
@@ -0,0 +1,256 @@
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications (no mesh refinement)
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications and adiabatic/shear heating specification
+# Velocity: free surface on top, outflow on sides, inflow at base
+# Temperature: fixed at top and bottom, insulating sides
+subsection Model settings
+  set Include adiabatic heating               = false
+  set Include shear heating                   = false
+  set Fixed temperature boundary indicators   = bottom, top
+  set Prescribed velocity boundary indicators = left x: function, right x:function, bottom y:function
+  set Free surface boundary indicators        = top
+end
+
+# Advecting the free surface vertically rather than
+# in the surface normal direction can result in a
+# more stable mesh when the deformation is large
+subsection Free surface
+  set Surface velocity projection = vertical
+end
+
+# Velocity on boundaries characterized by functions
+# The outward velocity (x-direction) on the left and right walls is 0.25 cm/year
+# The vertical velocity at the base is 0.125 cm/year (balances outflow on sides)
+# Velocity components parallel to the base (x-velocity) and side walls (y-velocity)
+# are unconstrained (i.e. 'free'). 
+subsection Boundary velocity model
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = m=0.0025, year=1
+    set Function expression =  if (x<200e3 , -1*m/year, 1*m/year) ; 0.00125
+  end
+end
+
+
+# Number and name of compositional fields
+# The four compositional fields represent the upper crust, lower crust, mantle
+# and a 'seed' placed in the mantle to help localize deformation.
+subsection Compositional fields
+  set Number of fields = 4
+  set Names of fields = upper, lower, mantle, seed
+end
+
+
+# Spatial domain of different compositional fields
+# The upper crust, lower crust and mantle are continuous horizontal layers
+# of varying thickness.  The top of the seed (4x8 km) is placed 2 km beneath
+# the base of the crust and straddles the horizontal midpoint.
+subsection Compositional initial conditions
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function expression = if(y>=80.e3, 1, 0); \
+                              if(y<80.e3 && y>=70.e3, 1, 0); \
+                              if(y<70.e3 && y>-100.e3,1, 0); \ 
+                              if(y<68.e3 && y>60.e3 && x>=198.e3 && x<=202.e3 , 1, 0);
+  end
+end
+
+# Temperature boundary conditions
+# Top and bottom (fixed) temperatures are consistent with the initial temperature field
+subsection Boundary temperature model
+  set Model name = box
+
+  subsection Box
+    set Bottom temperature = 1573
+    set Left temperature   =  273
+    set Right temperature  =  273
+    set Top temperature    =  273
```

Comment:
That was my reasoning as well and the temperature along the sides is basically governed by the top and bottom boundary conditions.  However, perhaps it would be more intuitive to specify the boundary conditions as being the same as the initial conditions even though it should have no effect on the final solution?